src/alert-manager/deploy/alert-templates/pai.tmpl

```diff
@@ -43,7 +43,11 @@ SOFTWARE.
       <div style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 14px; max-width: 600px; display: block; margin: 0 auto; padding: 0;">
         <table width="100%" cellpadding="0" cellspacing="0" style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 14px; border-radius: 3px; background-color: #fff; margin: 0; border: 1px solid #e9e9e9;" bgcolor="#fff">
           <tr>
+            {{ if gt (len .Alerts.Firing) 0 }}
```

Comment:
please double check firing and resolving won't be in the same mail.

Reply:
Yes, I've already checked.